.github/workflows/release.yaml

```diff
@@ -86,3 +108,7 @@ jobs:
           docker image tag tracee:latest aquasec/tracee:${ARCH}-${TAG}
           docker image push aquasec/tracee:${ARCH}
           docker image push aquasec/tracee:${ARCH}-${TAG}
+      - name: Sign Docker image
+        run: |
+          cosign sign -y $(docker inspect --format='{{index .RepoDigests 0}}' aquasec/tracee:${ARCH})
```

Comment:
You should use the "$IMAGE@$DIGEST" format to sign your container image, so the correct form is the following:

```
cosign sign -y aquasec/tracee:${ARCH}@$(docker inspect --format='{{index .RepoDigests 0}}' aquasec/tracee:${ARCH})
```

Reply:
oh, that's great, didn't notice this! Then, it looks correct, yes!